### PR TITLE
Chapter 6

### PR DESCRIPTION
**Pull request by thuongnn-0898**

### CODE REVIEW

**Comment by thuongnn-0898** [comment]
ready

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (VuThiTranVan) [inline app/helpers/static_pages_helper.rb]: Xóa file

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class UserTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
```
Comment (VuThiTranVan) [inline test/models/user_test.rb]: Xóa file

**Comment by VuThiTranVan** [comment]
merged
fix comment rồi mợt hì

**Pull request merged**